tests/airgapped/knative/README.md

```diff
@@ -0,0 +1,37 @@
+# Testing Knative in airgapped
+
+This directory is dedicated to testing Knative in an airgapped environment.
+
+## Prerequisites
+
+Prepare the airgapped environment and Deploy CKF by following the steps in [Airgapped test scripts](../README.md).
```

Comment:
```suggestion
Prepare the airgapped environment and deploy CKF by following the steps in [Airgapped test scripts](../README.md).
```

Reply:
Also, maybe we should link [this](https://github.com/canonical/bundle-kubeflow/tree/main/tests/airgapped#running-the-tests)?

Reply:
I linked the top-level heading so as to include the setup